src/components/shared/ChipList/index.tsx

```diff
@@ -1,12 +1,20 @@
 import { Box } from '@mui/material';
-import { useMemo, useState } from 'react';
+import { useEffect, useMemo, useRef, useState } from 'react';
 import { useIntl } from 'react-intl';
 import { ChipListProps } from './types';
 import ChipWrapper from './Wrapper';
 
-function ChipList({ values, disabled, maxChips, stripPath }: ChipListProps) {
+function ChipList({
+    values,
+    disabled,
+    maxChips,
+    stripPath,
+    sx,
+}: ChipListProps) {
     const intl = useIntl();
 
+    const listScroller = useRef<any>(null);
```

Comment:
Adding a scroller ref so that we can scroll the list slightly down when opened to make it clear it is scrollable.